reana/reana_dev/git.py

```diff
@@ -393,6 +430,7 @@ def git_clone(user, component, exclude_components):  # noqa: D301
                 run_command(cmd, component)
 
 
+
```

Comment:
Thanks for the PR! Please also add yourself to `AUTHORS.rst` in alphabetical order.  If you don't have ORCID yet, please create one on https://orcid.org/

Reply:
Also, please change the commit headline to something like `cli: improved git-clone first time behaviour`.  You can also mention in the commit log message `Closes #123` to indicate which issue number this commit closes.

Reply:
I have already made the changes, and added myself to the `AUTHORS.rst`